srcpkgs/nodejs/template

```diff
@@ -32,10 +35,11 @@ provides="nodejs-runtime-0_1"
 do_configure() {
 	local _args
 
-	export LD="$CXX"
+	export LD="$CC"
+	export LDXX="$CXX"
```

Comment:
LDXX ?

Reply:
Yup, node uses that too, e.g. here: https://github.com/nodejs/node/blob/8b4af64f50c5e41ce0155716f294c24ccdecad03/tools/gyp/pylib/gyp/generator/ninja.py

I guess we don't need it for `make` though right now.